Launch blog 4 (#67)
mintlify/mintlify.com: src/pages/blog/launch-week-day-4/index.mdx

import { ArticleLayout } from '@/components/blog/ArticleLayout'

export const meta = {
  title: 'Launch Week Day 4: API Playground Redesign',
  description: ' Redesigned API Playground for a new and improved user experience',
  mobileImageUrl: '/static/images/blog/launch-week-day-4/thumbnail.png',
  desktopImageUrl: '/static/images/blog/launch-week-day-4/square-api.png',
  author: 'Ronan McCarter',
  authorRole: 'Engineering',
  date: 'September 28, 2023',
  categories: ['Featured', 'Product'],
  authorImageUrl: '/static/images/blog/authors/ronan.png',
  readingTime: '2 minute read',
}

export default (props) => <ArticleLayout meta={meta} {...props} />

<img
  src="/static/images/blog/launch-week-day-4/thumbnail.png"
  alt="blog thumbnail"
  className="w-full rounded-md"
/>

Day 4 of Launch Week brings a redesigned API Playground to your docs.

> API specification is notoriously complex, and so are the tools used to document them. And as if creating API spec docs tools weren’t hard enough already, imagine the difficulties of building on with interactive and user-friendly UI. 

Reviewer: ```suggestion
APIs are notoriously complex, and so are the tools used to document them. And as if creating API-spec-docs-tools weren’t hard enough already, imagine the difficulties of building an interactive and user-friendly UI. 
```

Fortunately, our team is always eager for a good challenge. While many existing solutions are cumbersome to use, limited in function, and utterly ancient-looking, our API pages offer a competitive alternative with the help of our API Playground.  

> We don’t stop there. We wanted our API pages to be a standard-setter for the industry—powerful, easy to use, and amazing to look at. So, we overhauled our API Playground for a new and improved design.  

Reviewer: ```suggestion
We don’t stop there. We wanted our API pages to be a standard-setter for the industry—powerful, easy to use, and amazing to look at. So, we overhauled our API Playground with a new and improved design.  
```

The result? 

An upgraded API Playground that handles all edge cases and offers expanded features, including supercharged technical functions, effortless UX, and sleek UI.

## Why it matters

On average, half of all active customers use our API pages, of which the API Playground is one of the most important features offered. It allows users to craft API requests and send them from within the browser, without having to open a new window or write any code.
    
Given the API Playground’s role as one of our core offerings, it was our top priority for any upgrade.
 
Rather than taking a patchwork approach to target individual functions, we took a longer-term solution—completely revamping the API Playground for a new and improved user experience. 

Our mission to deliver the best docs-building platform continues, and we are proud to have you all with us on this journey.  

## What changed 

<video className="w-full" controls>
  <source src="/static/images/blog/launch-week-day-4/video-1.mp4" type="video/mp4" />
</video>

<video className="w-full" controls>
  <source src="/static/images/blog/launch-week-day-4/video-2.mp4" type="video/mp4" />
</video>

> An in-depth look at all the improvements we made—

Reviewer: ```suggestion
A high-level look at all the improvements we made—
```

> ### Technical Functionality
> - Wider range of input displays - better handling of more complex types such as nested arrays, objects, and mixed types
> - Improved handling of multimedia response types (image and audio files). 
> - Display of the status code and headers from the response
> 
> ### UI/UX
> - More accessible and intuitive input field UI
> - Improved error handling - provides rationale for request failure

Reviewer: ```suggestion
### Technical Functionality
- A wider range of input displays - first-class handling of more complex types such as nested arrays, objects, and mixed types
- Multimedia response handling - view images and stream audio responses directly in your docs

### UI/UX
- An all-around facelift
- More accessible and intuitive input field UI
- Exposing more response data to the user
```

## Getting started

> All API Playground updates are automatic, so no action needed 😀

Reviewer: ```suggestion
If you use OpenAPI to host your API docs on Mintlify, your docs will be upgraded automatically 😀

We'll be rolling out support for APIs defined with MDX in the coming weeks.
```
